packages/malloy/src/malloy.ts

```diff
@@ -691,6 +691,11 @@ export class Runtime {
     });
   }
 
+  // used internally by the testing framwork.
+  public _makeModelFromModelDef(_modelDef: ModelDef): ModelRuntimeRequest {
```

Comment:
@christopherswenson can you implement this?  Uncommenting out the calls in the describe section of  expr.spec.ts  should make this work.